lib/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Entitlements
   module Version
-    VERSION = "0.3.0"
+    VERSION = "0.4.0"
```

Comment:
I don't think we should cut a new version for this. The gem updates are essentially only for development of the plugin. The changes shouldn't impact downstream use of the gem, should it?